### ISSUE TITLE
Depend on static assets (as opposed to CommonJS modules)

### ISSUE THREAD

**Issue by julienrf** [scalacenter/scalajs-bundler#43]
Some NPM packages include static assets (fonts, stylesheets or images) in addition to CommonJS modules.

We should provide a way to use them from Scala.js projects.

**Comment by julienrf** [comment]
In the case of the WebScalaJSBundlerPlugin, it could be simple to introduce a new sbt key defining a set of resources to turn into assets: we could just add them to `sourceGenerators in Assets`.

**Comment by mathieuleclaire** [comment]
Nice ! Did you deployed it yet ? The 0.4.0-SNAPSHOT does not exist.

**Comment by julienrf** [comment]
Not yet, I’d like to fix #19 before making another release.

**Comment by mathieuleclaire** [comment]
npmAssets is not found in the 0.4.0-SNAPSHOT. What import is supposed to be used ?

**Comment by julienrf** [comment]
It’s available only in the `WebScalaJSBundler` plugin. Maybe it does not fit with your use case… Can you tell me more about your project?

**Comment by mathieuleclaire** [comment]
I didn't realised there was two differents plugins. Now I get it with 0.4.0-SNAPSHOT.
My project is still the https://github.com/openmole/scaladget project. It is a lib through which I bring already some js libs to a any other code thanks the ScalaJSBundlerPlugin. I'd like to bring also some css or fonts. Is it the right scope ?

**Comment by julienrf** [comment]
Indeed it is. But once I’ll have fixed #19 it will be slightly easier for you to achieve what you want.

For now, you probably want to add something similar to the following to the definition of the `runDemo` task:

~~~ scala
IO.copy(crossTarget.value / "node_modules" / "your-asset", demoTarget / "your-asset")
~~~

**Comment by mathieuleclaire** [comment]
Yes indeed. So I wait for the next issue fix ! Thanks.
